addons/website_slides_survey/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.website.slides.survey</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="website_slides.res_config_settings_view_form"/>
+        <field name="priority">10</field>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='website_slides_install_mass_mailing_slides']" position="after">
+                <group>
+                    <div class="col-12 col-lg-6 o_setting_box" id="elearning_install_certif">
```

Comment:
`id=website_slide_install_website_slides_survey` ?
(so we are consistent with `website_slides_install_mass_mailing_slides`)